specifications/active/aRMT-1.0.0.yml

```diff
@@ -6,6 +6,46 @@ version: 1.0.0
 assessment_type: QUESTIONNAIRE
 doc: aRMT Questionnaires definition. Includes Personal Health Questionnaire Depression Scale (PHQ-8), Experience sampling method (ESM) and RSES.
 data:
+  - type: THINC-IT
+    topic: notification_thinc_it
+    value_schema: .active.notification.Notification
+    questions:
+      - lead: Time for Thinc-it
+        content: It's time to complete the Thinc-it app.
+        widget: INFO
+  - type: ROMBERG_TEST
+    topic: task_romberg_test
+    value_schema: .active.task.Task
+    questions:
+      - lead: Romberg's Test
+        content: It's time to complete the Thinc-it app.
+        widget: INFO
```

Comment:
Please add a new enum for "INFO" widget in `/commons/catalogue/radar_widget.avsc`

Reply:
Done.